js/timepicker.js

```diff
@@ -485,6 +495,65 @@
       }
     }
 
+    _inputFromTextField() {
+      var isHours = this.currentView === 'hours';
+      var unit = Math.PI / (isHours ? 6 : 30);
+      var radius, radian;
```

Comment:
prefer the use of `const` (and `let` for the variables that are reassigned) instead of `var`

Reply:
Totally agree. I will update accordingly